docs/content/patterns/conditional-reveal/index.mdx

```diff
@@ -1,25 +1,25 @@
 ---
-title: Conditionally revealed questions
-description: Conditionally reveal a related question when a user selects a specific radio or checkbox option. This ensures that users only encounter questions when it is applicable to their selection.
+title: Conditionally revealed content
+description: Conditionally reveal a question or information related to a specific radio or checkbox option when a user selects it. This ensures that users only encounter conditionally revealed content when it is applicable to their selection.
 relatedComponents: ['radio', 'checkbox']
 ---
 
 <DoHeading />
 
-- only reveal 1 related question
-- reveal the related question directly under the initial question
-- use another page if there are multiple related questions
+- limit what is revealed to 1 related question or piece of information
+- reveal the related question or information directly under the initial question
+- use another page or drawer if there are multiple related questions
```

Comment:
Actually this feels weird. How would a user navigate to a drawer?

Reply:
agreed, please remove or drawer